### PR TITLE
Giane/[01] Refactor: migrate to androidx

### PR DESCRIPTION
**Pull request by gianebinotto**
Migrate lib core to androidx

### CODE REVIEW

File: app/src/androidTest/java/com/rubenshardt/cheesecakecoreexample/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -18,7 +17,7 @@ class ExampleInstrumentedTest {
     @Test
     fun useAppContext() {
         // Context of the app under test.
-        val appContext = InstrumentationRegistry.getTargetContext()
+        val appContext = androidx.test.InstrumentationRegistry.getTargetContext()
```
Comment (guilucas07) [inline app/src/androidTest/java/com/rubenshardt/cheesecakecoreexample/ExampleInstrumentedTest.kt]: add this import at the top of file, like "import androidx.test.ext.junit.runners.AndroidJUnit4"

File: core/src/androidTest/java/io/cheesecakelabs/core/ExampleInstrumentedTest.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
     @Test
     public void useAppContext() {
         // Context of the app under test.
-        Context appContext = InstrumentationRegistry.getTargetContext();
+        Context appContext = androidx.test.InstrumentationRegistry.getTargetContext();
```
Comment (guilucas07) [inline core/src/androidTest/java/io/cheesecakelabs/core/ExampleInstrumentedTest.java]: add this import at the top of file, like "import androidx.test.ext.junit.runners.AndroidJUnit4"

File: retrofit/src/androidTest/java/io/cheesecakelabs/retrofit/ExampleInstrumentedTest.java [hunk 3]
```diff
@@ -19,7 +18,7 @@
     @Test
     public void useAppContext() {
         // Context of the app under test.
-        Context appContext = InstrumentationRegistry.getTargetContext();
+        Context appContext = androidx.test.InstrumentationRegistry.getTargetContext();
```
Comment (guilucas07) [inline retrofit/src/androidTest/java/io/cheesecakelabs/retrofit/ExampleInstrumentedTest.java]: add this import at the top of file, like "import androidx.test.ext.junit.runners.AndroidJUnit4"
Are these files necessary?

**Review by guilucas07 — CHANGES_REQUESTED** [review]

**Review by diegohkd — CHANGES_REQUESTED** [review]
Same as Gui

File: retrofit/src/androidTest/java/io/cheesecakelabs/retrofit/ExampleInstrumentedTest.java (hunk 3 above)
Comment (gianebinotto) [reply retrofit/src/androidTest/java/io/cheesecakelabs/retrofit/ExampleInstrumentedTest.java]: I just kept them because they already existed.

**Review by diegohkd — APPROVED** [review]

File: retrofit/src/androidTest/java/io/cheesecakelabs/retrofit/ExampleInstrumentedTest.java (hunk 3 above)
Comment (guilucas07) [reply retrofit/src/androidTest/java/io/cheesecakelabs/retrofit/ExampleInstrumentedTest.java]: so, can be removed, right?

Comment (gianebinotto) [reply retrofit/src/androidTest/java/io/cheesecakelabs/retrofit/ExampleInstrumentedTest.java]: Done

**Review by guilucas07 — APPROVED** [review]

**Review by RubensHardt — APPROVED** [review]

**Pull request merged**